content/hardware/05.pro-solutions/solutions-and-kits/portenta-machine-control/product.md

```diff
@@ -2,12 +2,13 @@
 title: Portenta Machine Control
 url_shop: https://store.arduino.cc/portenta-machine-control
 hasSetupGuide: true
-primary_button_url: /tutorials/portenta-machine-control/user-manual
-primary_button_title:  Arduino User Manual
+primary_button_url: /software/plc-ide/tutorials/plc-ide-setup-license
+primary_button_title:  PLC IDE License
```

Comment:
@Matteo-it this is not matching the main branch :( it should be the user manual

Reply:
We just need to make sure your branch is updated to main before merging ⭐

Reply:
I've not rebased my branch on main, i'll do it just before merging, so for now it's not updated!